presto-iceberg/src/main/java/com/facebook/presto/iceberg/IcebergPageSinkProvider.java

```diff
@@ -51,14 +51,14 @@ public IcebergPageSinkProvider(
             JsonCodec<CommitTaskData> jsonCodec,
             IcebergFileWriterFactory fileWriterFactory,
             PageIndexerFactory pageIndexerFactory,
-            IcebergConfig config)
+            IcebergConfig icebergConfig)
     {
         this.hdfsEnvironment = requireNonNull(hdfsEnvironment, "hdfsEnvironment is null");
         this.jsonCodec = requireNonNull(jsonCodec, "jsonCodec is null");
         this.fileWriterFactory = requireNonNull(fileWriterFactory, "fileWriterFactory is null");
         this.pageIndexerFactory = requireNonNull(pageIndexerFactory, "pageIndexerFactory is null");
-        requireNonNull(config, "config is null");
-        this.maxOpenPartitions = config.getMaxPartitionsPerWriter();
+        requireNonNull(icebergConfig, "icebergConfig is null");
+        this.maxOpenPartitions = icebergConfig.getMaxPartitionsPerWriter();
```

Comment:
This change should be in the first commit.

Reply:
Moved. Thanks.